test/system/250-systemd.bats

```diff
@@ -281,6 +281,15 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
     is "$output" "" "output should be empty"
 }
 
+@test "podman --systemd sets container_uuid" {
+    run_podman run --systemd=always --name test $IMAGE printenv container_uuid
+    uuid=$output
+    run_podman inspect test --format '{{ .ID }}'
+    run expr substr "${output}" 1 32
+    is "$uuid" "$output" "UUID should be first 32 chars of Container id"
+
```

Comment:
Suggestion:
```
    run_podman inspect ...
    is "$uuid" "${output:0:32}" "container_uuid envariable should be the first 32 chars of container ID"
    run_podman rm test
```

Reply:
I figured you'd have an easier way.

Reply:
I find the use of `UUID` here (in the comment) misleading, that's why I suggested "container_uuid envariable" instead.

Also, could you remove the unneeded trailing empty line?

Also, this seems the sort of thing that needs documenting, I just can't find where it should go